src/cryptenroll/cryptenroll-password.c

```diff
@@ -27,8 +28,7 @@ int enroll_password(
                 if (!new_password)
                         return log_oom();
 
-                string_erase(e);
```

Comment:
I don't follow, we don't need to do the erase thing anymore?

Reply:
ie, the content of "e" is now left behind?

Reply:
That's the whole point of this PR. e == getenv("NEWPASSWORD") here, so it'll get erased in unsetenv_erase. Afaik, getenv() points to some global area.

Reply:
ah I missed that it was all globals